src/mumble/MainWindow.cpp

```diff
@@ -524,8 +525,12 @@ void MainWindow::closeEvent(QCloseEvent *e) {
 			showMinimized();
 			e->ignore();
 
-			if (qcbRemember->isChecked()) {
-				Global::get().s.connectedQuitBehavior = ConnectedQuitBehavior::MINIMIZE;
+			/* If checkbox is checked and not connected, keep asking when connected
+			 * If checkbox is checked and connected, always minimize when connected */
```

Comment:
```suggestion
			// If checkbox is checked and not connected, keep asking when connected
			 // If checkbox is checked and connected, always minimize when connected
```
This doesn't seem to really warrant a multiline comment :D